src/core/server/ui_settings/ui_settings_client.ts

```diff
@@ -62,6 +64,9 @@ type UiSettingsRawValue = UiSettingsParams & UserProvidedValue;
 type UserProvided<T = unknown> = Record<string, UserProvidedValue<T>>;
 type UiSettingsRaw = Record<string, UiSettingsRawValue>;
 
+// identifier for current user
+export const CURRENT_USER = '<current_user>';
```

Comment:
```suggestion
export const CURRENT_USER_PLACEHOLDER = '<current_user>';
```
`CURRENT_USER` seems a little bit confusing to me, actually it is a placeholder if I understand correctly.

Reply:
make sense, will updated to suggested name.